AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -772,7 +775,7 @@ - (void)processLogBufferAfterCrash {
 
           // Buffered logs are used sending their own channel. It will never contain more than 50 logs.
           MSLogDebug([MSCrashes logTag], @"Re-enqueueing buffered log, type: %@.", item.type);
-          [self.bufferChannelUnit enqueueItem:item];
+          [self.bufferChannelUnit enqueueItem:item flags:MSFlagsNone];
```

Comment:
Should it be `Normal`?

Reply:
None needs to map to default values for each flags as discussed.

Reply:
Oh wait crashes should be critical actually.